fff

```diff
@@ -368,6 +369,57 @@ trash() {
     fi
 }
 
+bulk_rename() {
+  local tmp_file_template="${XDG_CACHE_HOME:=${HOME}/.cache}/fff/bulkrename.XXXXX"
+
+  local tmp_file
+  printf -v tmp_file '%q' "$(mktemp "$tmp_file_template")"
+
+  # Write all input file names to temporary file and open it in an editor.
+  printf "%s\n" "${@:1:$#-1}" > "$tmp_file"
+  "${EDITOR:-vi}" "$tmp_file"
+
+  local commands_file
+  printf -v commands_file '%q' "$(mktemp "$tmp_file_template")"
+
+  # Add instructions to the file with commands.
+  printf '%s\n%s\n' \
+    "# This file will be executed when you close the editor." \
+    "# Please double-check everything, clear the file to abort." > "$commands_file"
```

Comment:
This line is too long. You can either split the `> file` to a new line (same as the above strings) or change the message to something shorter.